# Medidas de Tendencia Central y Percentiles

## Objetivo:

Aplicar las medidas de tendencia central (media, mediana y moda) y medidas de posición no central (percentiles) en el análisis de datos, utilizando Python como herramienta de apoyo.

*Nota: Este laboratorio está diseñado para desarrollar razonamiento estadístico. Evita el uso de herramientas de IA para responder las preguntas a menos que esto sea explícitamente sugerido* 🤖. *En su lugar, enfócate en aplicar los conceptos revisados.*

## Actividad 1

La siguiente base de datos posee los registros de todas las calificaciones obtenidas por los alumnos del segundo semestre del 2019, jornada diurna en la asignatura de Álgebra.

Para responder las siguientes preguntas, descarga la base de datos **<a href="https://drive.google.com/uc?export=download&id=10tm66MrnBpkR1_aY10yxEAudwirzDcJK
">aquí</a>**.

In [3]:
import pandas as pd
df_1 = pd.read_csv(r"Actividad_1.csv")
df_1

ID Alumno  Taller 1  Taller 2  Taller 3  Actividad en línea 1  \
0    160001322       5.5       7.0       5.6                   6.4   
1    160002042       7.0       7.0       7.0                   3.8   
2    160002581       6.5       6.2       4.5                   5.4   
3    160001794       7.0       7.0       6.5                   7.0   
4    160003667       6.0       4.3       6.0                   4.8   
..         ...       ...       ...       ...                   ...   
511  160002897       7.0       7.0       7.0                   6.6   
512  160000303       7.0       7.0       6.5                   7.0   
513  160001258       5.7       6.0       6.8                   7.0   
514  160003661       7.0       7.0       7.0                   6.6   
515  160003452       7.0       7.0       6.8                   6.2   

     Actividad en línea 2  Actividad en línea 3  Prueba 1  Prueba 2  Prueba 3  \
0                     6.2                   4.6       6.2       5.5       4.3   
1                     2.3                   2.3       5.0       4.9       4.4   
2                     1.8                   2.4       6.2       5.1       4.0   
3                     5.3                   2.6       6.2       7.0       6.3   
4                     4.4                   3.0       5.3       4.7       4.2   
..                    ...                   ...       ...       ...       ...   
511                   5.4                   6.2       4.6       6.1       4.2   
512                   7.0                   3.8       4.4       4.7       4.0   
513                   6.5                   4.0       6.0       5.1       3.1   
514                   6.5                   5.7       6.4       6.6       6.2   
515                   4.9                   2.4       6.1       3.1       3.0   

     Nota Presentación  Nota Examen  Nota Final  
0                  5.5          6.2         5.8  
1                  4.8          5.2         5.0  
2                  5.0          4.7         4.9  
3                  6.4          6.6         6.5  
4                  4.7          5.8         5.1  
..                 ...          ...         ...  
511                5.5          4.2         5.0  
512                5.0          4.6         4.8  
513                5.1          6.5         5.7  
514                6.5          6.3         6.4  
515                4.5          4.9         4.7  

[516 rows x 13 columns]

1. ¿Cuál es la población en estudio?

La población de estudios son los Alumnos de Algebradel segundo semestre 2019.as_integer_ratio

2. ¿Cuál es la muestra en estudio?

La muestra son los 516 alumnos

3.  Identifique y clasifique las variables involucradas en el estudio.

    Las 12 variables estadisticas son CUANTITATIVAS CONTINUAS porque son valores calculadas.

4. Calcule e interprete las medidas de tendencia central de las notas del examen.

In [5]:
media = df_1["Nota Examen"].mean()
moda = df_1["Nota Examen"].mode()
mediana = df_1["Nota Examen"].median()

print(f"El promedio: {media:.1f}\nmoda: {moda}\nmediana: {mediana:.1f}")

El promedio: 5.1
moda: 0    5.3
Name: Nota Examen, dtype: float64
mediana: 5.2


5. ¿Entre qué valores se encuentra el 60% de las notas más bajas en la prueba 1?

In [6]:
minimo_prueba1 = df_1["Prueba 1"].min()
percentil_60 = df_1["Prueba 1"].quantile(0.60)

print(f"El 60% de las notas mas bajas de la prueba 1 se encuentran entre: {minimo_prueba1} y {percentil_60}")

El 60% de las notas mas bajas de la prueba 1 se encuentran entre: 1.6 y 5.7


6.  La institución de educación creará cursos de reforzamiento para todos los estudiantes cuyas calificaciones en la Prueba 2, estén en el 25% más bajo. ¿Cuál es la nota más alta dentro de este grupo que va a reforzamiento?

In [7]:
percentil_25 = df_1["Prueba 2"].quantile(0.25)

print(f"La nota más alta del 25% mas baja en la Prueba 2 es: {percentil_25}")

La nota más alta del 25% mas baja en la Prueba 2 es: 4.4


7. ¿Cuál es la nota más baja entre el 30% de mejores notas del examen?

In [8]:
percentil_70 = df_1["Nota Examen"].quantile(0.70)

print(f"La nota más baja de los mejores 30% es de: {percentil_70}")

La nota más baja de los mejores 30% es de: 5.7


8. Si se desea reevaluar al 10% más bajo en el examen, ¿cuál es rango de notas que se debe considerar?

In [ ]:
percentil_10 = df_1["Nota Examen"].quantile(0.10)
mínimo_examen = df_1["Nota Examen"].min()

print(f"Lo nota más baja del 10% del examen es: {mínimo_examen} ")

Lo nota más baja del 10% del examen es: 3.5 


9. Si la institución va a premiar al 5% de las mejores notas finales, ¿cuál es la nota más baja registrada por ese grupo?

## Actividad 2

Imagina que trabajas como analista de datos en una consultora de orientación vocacional que asesora a estudiantes de enseñanza media sobre qué carrera elegir. Recibes una base de datos con información de carreras técnicas y profesionales de distintas instituciones chilenas de los años 2024 y 2025. Tu misión es analizar los ingresos y empleabilidad de los titulados, y entregar conclusiones claras que los ayuden a tomar decisiones.

**Utiliza la siguiente base de datos para responder las actividades 2, 3 y 4.**

Descarga la base de datos **<a href="https://docs.google.com/spreadsheets/d/1O7lb2OqNCSnqJaswSNqmD4HuI0BWfcsP/export?format=xlsx
">aquí</a>**.

1. Identifica la población y muestra del estudio.

2. Clasifica las variables presentes en la base de datos.

3. ¿Cuál es el ingreso promedio de los titulados? ¿Y la mediana? ¿Qué puedes concluir de la diferencia entre ambos?

4. ¿Cuál es la moda de los ingresos? Interprétala.

5. Calcula los percentiles 25, 50 y 75 del ingreso de los titulados e interprétalos.

6. Utiliza la función *describe()* de Pandas para validar tus resultados.

7. Considera que tres personas desean estudiar Técnico en enfermería, pero cada una valora cosas distintas:
* Sofía busca maximizar su ingreso futuro.
* Diego prioriza encontrar trabajo rápidamente.
* Camila quiere un equilibrio entre ambas.

$\hspace{1cm}$Considerando la media como medida representativa, ¿qué tipo de institución recomendarías a cada uno y por qué?

## Actividad 3

Un grupo de estudiantes está evaluando si el nivel de acreditación de una institución podría impactar sus resultados laborales. Para responder esta pregunta, realiza el siguiente análisis:

**Utiliza la siguiente base de datos para responder las actividades 2, 3 y 4.**

Descarga la base de datos **<a href="https://docs.google.com/spreadsheets/d/1O7lb2OqNCSnqJaswSNqmD4HuI0BWfcsP/export?format=xlsx
">aquí</a>**.

1. Agrupa los datos por años de acreditación y calcula la media y la mediana del ingreso al 4to año de cada grupo.

2. ¿Existe una relación aparente entre la cantidad de años de acreditación y el ingreso promedio? ¿Qué patrón observas?

3. ¿Se observa una tendencia similar al analizar la empleabilidad al primer año?

4. Comparando las variables ingreso y empleabilidad, analiza cómo varían la media y la mediana en cada caso. ¿Qué conclusiones generales puedes extraer sobre la representatividad de estas medidas?

5. Teniendo en cuenta tu análisis sobre cómo varían los ingresos y la empleabilidad con los años de acreditación, ¿qué consejo le darías a un estudiante que va a elegir una institución basándose únicamente en el nivel de acreditación? Justifica tu recomendación.

## Actividad 4

Imagina que trabajas como analista del Ministerio de Educación. Te han solicitado caracterizar los resultados laborales de los titulados por área de formación utilizando una base de datos con información de carreras técnicas y profesionales de distintas instituciones chilenas de los años 2024 y 2025.

**Utiliza la siguiente base de datos para responder las actividades 2, 3 y 4.**

Descarga la base de datos **<a href="https://docs.google.com/spreadsheets/d/1O7lb2OqNCSnqJaswSNqmD4HuI0BWfcsP/export?format=xlsx
">aquí</a>**.

1. ¿Cuál Área se observa con mayor frecuencia entre los titulados?

2. ¿Entre qué sueldos se encuentran los egresados del 10% más alto al 4to año de titulación?

3. ¿Entre qué sueldos se encuentran los egresados del 17% más bajo al 4to año de titulación?

4. Agrupa los datos por Área y para cada grupo calcula las siguientes medidas del Ingreso al 4° año:
* Media
* Mediana
* Percentil 25
* Percentil 75

$\hspace{1cm}$Luego, construye un DataFrame que reúna estas cuatro columnas.

5. Interpreta las medidas calculadas en las áreas de Agropecuaria y Tecnología.

6. Al comparar la media con la mediana en el Área Salud se observa que la media es mayor. ¿Qué característica de los datos podría explicar esta diferencia?

7. Describe lo que ocurre con la media del ingreso al 4to año de titulación, según Área y Tipo de institución. ¿Qué conclusiones puedes extraer acerca del impacto del tipo de institución en el ingreso según área?

*Sugerencia: Puedes crear una tabla bivariada utilizando la función crosstab() de Pandas*.